jre/kotlin/AmpliApp/src/main/kotlin/com/amplitude/ampli/Ampli.kt

```diff
@@ -749,6 +719,42 @@ open class Ampli {
         this.track(userId, event, options, extra)
     }
 
+    /**
```

Comment:
This file still shows "Identify" https://github.com/amplitude/ampli-examples/blob/3b643ec6f7d77c46ed66663a465c696f6f454c32/jre/kotlin/AmpliApp/src/main/kotlin/com/amplitude/ampli/Ampli.kt#L66

Reply:
This PR doesn't contain identify-related changes - it only updates outdated generated files to simplify next diffs in https://github.com/amplitude/ampli-examples/pull/219.